modules/social_features/social_event/src/Form/EnrollActionForm.php

```diff
@@ -187,10 +193,21 @@ public function buildForm(array $form, FormStateInterface $form_state) {
             break;
```

Comment:
It could be that this break is not properly handled for multi-group events and has potential to break the expected result. However this is out of scope of given bug.